src/dotnet/commands/dotnet-add/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.Build.Construction;
+using Microsoft.DotNet.ProjectJsonMigration;
+using NuGet.Frameworks;
+using Microsoft.DotNet.Tools.Add.ProjectToProjectReference;
+
+namespace Microsoft.DotNet.Tools.Add
+{
+    public class AddCommand
+    {
+        public const string CommandName = "dotnet-add";
+        public const string UsageText = @"Usage: dotnet add [command] [arguments]
+
+Arguments:
```

Comment:
It's really not clear to me why hand-parsing the arguments is a benefit here. The code length is ~= to the usage of the standardized parser, above, and it's another pattern for folks to learn/keep up to date. When we add `tab completion`, for example, we'll need to replace this implementation OR implement tab completion twice. If this does need to be unique because of a limitation in the parser we use everywhere else then please make sure that help text formatting, error handling [e.g. bad options], etc. are identical to how the standard parser does the same. Let's not bleed implementation details.